book/website/community-handbook/local-build.md

```diff
@@ -0,0 +1,67 @@
+(ch-local-build)=
+# How to build Jupyter Book locally
+
+## But why build locally?
+It's always handy to be able to preview any changes you have been working on as you go - you can be confident that changes you have made are accurate and as intended. 
+A nice way to do this is to use the underlying Jupyter Book tool to build the book locally.
+
+This is useful because it allows you to preview any changes you have made on your local machine *before* you push your changes to a remote branch. 
+You can then decide if you are happy with the result and push your changes to the remote branch thus helping to keep Pull Request conversations and commit histories a bit cleaner.
+
+## Step-by-step guide
+1. Install miniconda https://conda.io/projects/conda/en/latest/user-guide/install/index.html#regular-installation
+2. Run `conda init` in your terminal. You should see `(base)` in your prompt indicating that conda was successfully installed and you are now in it's base environment.
+3. Create a new environment and install a modern version of python into it
+   ```
+   conda create --name the-turing-way python=3.10
+   ```
+4. Activate the environment with `conda activate the-turing-way`. Any commands we run with Python or pip from now on will use the versions of Python and pip installed into _this_ conda env, not any others.
+5. Clone the repo: `git clone https://github.com/alan-turing-institute/the-turing-way`
+6. Change into the repo: `cd the-turing-way`
+7. Change into the correct folder `cd book/website`
+8. Install the dependencies _into your conda environment_
+   ```
+   pip install -r requirements.txt
+   ```
+9. And now build the book:
+   ```
+   jupyter-book build .
+   ```
+10. The output of the build process will provide output such as below that demonstrate how you can view the book locally:
+    ```
+    ===============================================================================
+
+    Finished generating HTML for book.
+    Your book's HTML pages are here:
+        _build/html/
+    You can look at your book by opening this file in a browser:
+        _build/html/index.html
+    Or paste this line directly into your browser bar:
+        file:///Users/sgibson/source/github/alan-turing-institute/the-turing-way/book/website/_build/html/index.html
+
+    ===============================================================================
+    ```
+    
+### Build the book while working on a Pull Request
+If you would like to preview a version of the book from a certain branch (perhaps to render the book while working on a PR) then simply switch to the required branch and rebuild the book as in step 10:
+   ```
+   git checkout mybranch
+   jupyter-book build .
+   ```
+Follow the link as before and you will see changes specific to that branch rendered.
+
+## Why did we recommend using miniconda?
```

Comment:
```suggestion
## Why did we recommend using (mini)conda?
```

Just to indicate that the package manager we are using is conda, and the distribution of conda we are using is miniconda.